Source/DataSources/ImageMaterialProperty.js

```diff
@@ -155,7 +136,6 @@ define([
         var color = Property.getValueOrUndefined(this._color, time, result.color);
         if (!defined(color)) {
```

Comment:
Replace this if block with

`result.color = Property.getValueOrClonedDefault(this._color, time, defaultColor, result.color);`